### PR TITLE
Take care to reset relative URL to prevent issues; fixes #783

### LINKED ISSUE #783: HTML report sometimes add extra parent path
Sometimes (I cannot really tell when it happens), some generated coverage HTML files have a wrong path. This affetcs the top link and the path to the CSS file as well (well, at least).

On a correct generated file, I'll have `src="screen.css"` but `src="../../screen.css"` on wrong ones.

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: classes/report/fields/runner/coverage/html.php [hunk 1]
```diff
@@ -116,6 +116,8 @@ public function __toString()
 
                     if (substr_count($className, '\\') >= 1) {
                         $classTemplate->relativeRootUrl = rtrim(str_repeat('../', substr_count($className, '\\')), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR;
+                    } else {
+                        $classTemplate->relativeRootUrl = '';
```
Comment (Hywan) [inline classes/report/fields/runner/coverage/html.php]: I think `'./'` might be a little bit more consistent than `''`. Thoughts?

**Comment by Hywan** [comment]
Thanks for the patch!

File: classes/report/fields/runner/coverage/html.php (hunk 1 above)
Comment (trasher) [reply classes/report/fields/runner/coverage/html.php]: It seems `relativeRootUrl` is empty by default; that's why I've used `''`. I've just tested with `./`; it works as well. Lemme know what you prefer, I'll change.

Comment (Hywan) [reply classes/report/fields/runner/coverage/html.php]: I prefer `./` :-).

Comment (trasher) [reply classes/report/fields/runner/coverage/html.php]: Changed

**Pull request merged**